controllers/searchoperator_controller.go

```diff
@@ -67,6 +68,7 @@ var (
 	storageClass         = ""
 	storageSize          = "10Gi"
 	namespace            = os.Getenv("WATCH_NAMESPACE")
+	setupPod, _          = strconv.ParseBool(os.Getenv("DEPLOY_REDISGRAPH"))
```

Comment:
We need to handle the case where this env variable is not set to reduce the risk of breaking downstream.

https://golang.org/pkg/os/#Getenv